### PR TITLE
ddt removal in ui usergroups tests

### PR DESCRIPTION
**Pull request by sthirugn**
Test results:
```sh
# nosetests tests/foreman/ui/test_usergroup.py
.
----------------------------------------------------------------------
Ran 6 tests in 37.036s

OK (skipped=5)
```

### CODE REVIEW

File: tests/foreman/ui/test_usergroup.py [hunk 1]
```diff
@@ -50,29 +47,15 @@ def test_negative_create_usergroup_with_too_long_name(self, group_name):
...
         """
         with Session(self.browser) as session:
-            make_usergroup(
-                session, org=self.organization.name, name=group_name)
-            self.assertIsNotNone(self.usergroup.wait_until_element(
-                common_locators['name_haserror']))
-            self.assertIsNone(self.usergroup.search(group_name))
-
-    @data('', ' ')
-    def test_negative_create_usergroup_with_blank_name(self, group_name):
```
Comment (sthirugn) [inline tests/foreman/ui/test_usergroup.py]: I combined this test and the previous one to create a new invalid test which uses invalid_values_list() helper.

File: tests/foreman/ui/test_usergroup.py [hunk 2]
```diff
@@ -34,14 +31,14 @@ def test_positive_create_usergroup(self, group_name):
...
-            self.assertIsNotNone(self.usergroup.search(group_name))
+            for group_name in generate_strings_list():
+                with self.subTest(group_name):
+                    make_usergroup(session, name=group_name, users=[user_name])
+                    self.assertIsNotNone(self.usergroup.search(group_name))
 
     @skip_if_bug_open('bugzilla', 1142588)
-    @data(*generate_strings_list(len1=256))
-    def test_negative_create_usergroup_with_too_long_name(self, group_name):
+    def test_negative_create_usergroup_with_too_long_name(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_usergroup.py]: maybe `def test_negative_create_usergroup` will be better to use now

**Comment by oshtaier** [comment]
pending small comment

File: tests/foreman/ui/test_usergroup.py (hunk 2 above)
Comment (sthirugn) [inline tests/foreman/ui/test_usergroup.py]: :+1:

**Comment by sthirugn** [comment]
Comments addressed.

**Comment by oshtaier** [comment]
ACK

File: tests/foreman/ui/test_usergroup.py [hunk 3]
```diff
@@ -80,18 +63,18 @@ def test_negative_create_usergroup_with_same_name(self, group_name):
         @Assert: Usergroup cannot be  created with existing name
 
         """
+        group_name = gen_string('alphanumeric')
         with Session(self.browser) as session:
-            make_usergroup(
-                session, org=self.organization.name, name=group_name)
+            make_usergroup(session, org=self.organization.name,
+                           name=group_name)
```
Comment (abalakh) [inline tests/foreman/ui/test_usergroup.py]: AFAIK we prefer regular indents over hanging code, long time ago we were updating all the hanging code occurences, and now you're reverting everything back? May I know the reason why?

**Comment by abalakh** [comment]
Pending question + rebase.

File: tests/foreman/ui/test_usergroup.py (hunk 3 above)
Comment (oshtaier) [inline tests/foreman/ui/test_usergroup.py]: Check answer in that PR
https://github.com/SatelliteQE/robottelo/pull/2928
Maybe that is same thing you are referring here

Comment (sthirugn) [inline tests/foreman/ui/test_usergroup.py]: For fewer parameters like this, I agree that hanging indents are better than hanging code.  I will fix this. 

But UI has some functions which take 10 arguments or so.  This is a code flaw that the function takes 10+ arguments which needs to be addressed.  Until it is addressed, IMO it is better to use hanging code just for those functions which is fewer lines of code vs. hanging indent.

**Comment by sthirugn** [comment]
Comments addressed and rebased

**Comment by abalakh** [comment]
ACK

**Comment by sthirugn** [comment]
part of https://github.com/SatelliteQE/robottelo/issues/2769

**Pull request merged**